basics/uattributes.adoc

```diff
@@ -226,3 +226,39 @@ a|The link:uri.adoc[UUri] that the service consumer expects to receive this resp
 == Validation
 
 Each link:../languages.adoc[uProtocol Language Library] *MUST* provide means to assert the consistency of a message's attributes according to its type as defined in the sections above. The concrete implementation should follow common practices of the particular programming language.
+
+
+== Listeners for different message types
```

Comment:
You're right. `up-l1/README.adoc` is a better place
Regarding the first table, I still think it's necessary, otherwise, it's not easy for newcomers to understand how we get the second table.